source/tutorials/pd-mice.rst

```diff
@@ -460,6 +408,17 @@ We can test our covariates of interest against Faith's phylogenetic diversity an
 
 If we had a continuous covariate that we thought was associated with the alpha diversity, we could test that using ``qiime diversity alpha-correlation``. However, the only continuous variable in this dataset is the ``days_since_transplant``.
 
+In some experiments, multiple interacting factors may impact alpha diversity together. If our alpha diversity estimates follow a normal distribution, we may use analysis of variance (ANOVA) to test whether multiple effects significantly impact alpha diversity. This test is present in the ``q2-longitudinal`` plugin:
+
+.. command-block::
+
+   qiime longitudinal anova \
+     --m-metadata-file ./core-metrics-results/faith_pd_vector.qza \
+     --m-metadata-file ./metadata.tsv \
+     --p-formula 'faith_pd ~ genotype * donor_status' \
+     --o-visualization ./core-metrics-results/faiths_pd_anova.qzv
+
```

Comment:
That's shiny. Although I never would have thought to look for it in q2-longitudinal. Good for the richness metrics which usually are asymptotically normal

Reply:
yeah @thermokarst and I had discussed back and forth about where to add it... I originally put it in q2-longitudinal thinking that it could be useful for simple longitudinal analyses and because it has many of the same dependencies and underlying functions that the LME visualizer uses. It just sort of stuck. Right now it is definitely hidden so having it in a tutorial like this would help make it more visible.

Reply:
actually one really neat thing that could go in that visualizer if you are interested is a test for normality.

Reply:
This gets back to the ongoing discussion about needing a diversity-specific tutorial that keeps coming up

Reply:
Definitely! One day...